playwright-ct.config.mts

```diff
@@ -3,5 +3,7 @@ import { defineConfig } from '@playwright/experimental-ct-react';
 import config from './playwright-base.config.mjs';
 
 config.testDir = './tests/components/';
-
+config.reporter = process.env.CI
+  ? 'html'
+  : [['html', { outputFolder: 'playwright-report-ct' }]];
```

Comment:
Dev: report to a specific directory so that `e2e` and `ct` don't overwrite each other (important for `npm run playwright`).
CI: each type of test runs independently so we output to the default folder.

Reply:
Can you still add the github reporter in CI?